randovania/games/prime1/layout/prime_configuration.py

```diff
@@ -60,6 +60,18 @@ class PrimeConfiguration(BaseConfiguration):
     qol_pickup_scans: bool
     qol_cutscenes: LayoutCutsceneMode
 
+    range_scale_low: float = dataclasses.field(metadata={"min": 0.01, "max": 25.0, "precision": 1.0})
```

Comment:
please make these all part of 1 dict or class under 1 config e.g. `enemy_stat_rando: ...`